micro_processors/EventDirectorProcessor_Proxy_Test.postman_collection.json

```diff
@@ -15,31 +15,31 @@
 						"exec": [
 							"responseJSON = pm.response.json();\r",
 							"\r",
-							"let outgoingCRSPResult = responseJSON.data;\r",
-							"let transaction = outgoingCRSPResult.transaction\r",
-							"let networkMap = outgoingCRSPResult.networkMap\r",
-							"let dataCache = outgoingCRSPResult.DataCache\r",
-							"let metaData = outgoingCRSPResult.metaData;\r",
-							"let prcgTm = outgoingCRSPResult.metaData.prcgTmCRSP;\r",
+							"let EveoutgoingEventDirectorResult = responseJSON.data;\r",
+							"let transaction = EveoutgoingEventDirectorResult.transaction\r",
+							"let networkMap = EveoutgoingEventDirectorResult.networkMap\r",
+							"let dataCache = EveoutgoingEventDirectorResult.DataCache\r",
+							"let metaData = EveoutgoingEventDirectorResult.metaData;\r",
+							"let prcgTm = EveoutgoingEventDirectorResult.metaData.prcgTmEventDirector;\r",
```

Comment:
Expecting prcgTmED instead of prcgTmEventDirector

Reply:
Resolved.